examples/petstore/controllers/pets.go

```diff
@@ -19,6 +19,8 @@ func (rs PetsRessources) Routes(s *fuego.Server) {
 	fuego.Get(petsGroup, "/{id}", rs.getPets)
 	fuego.Get(petsGroup, "/by-name/{name...}", rs.getPetByName)
 	fuego.Put(petsGroup, "/{id}", rs.putPets)
+	fuego.Put(petsGroup, "/{id}/json", rs.putPets).
+		RequestContentType("application/json")
```

Comment:
maybe 


```suggestion
		ExpectContentType("application/json")
```

or


```suggestion
		RequireContentType("application/json")
```

or


```suggestion
		AcceptContentType("application/json")
```

What do you think?

Reply:
Not sure I'm following?

We're applying the Content-Type for the openapi RequestBody.

Reply:
I assumed the method was expecting the caller to send this.

I might have misunderstood the PR purpose